packages/aws-cdk/test/commands/migrate.test.ts

```diff
@@ -177,7 +177,7 @@ describe('Migrate Function Tests', () => {
     // Replaced stack file is referenced correctly in app file
     const app = fs.readFileSync(path.join(workDir, 'GoodCSharp', 'src', 'GoodCSharp', 'Program.cs'), 'utf8').split('\n');
     expect(app.map(line => line.match('namespace GoodCSharp')).filter(line => line).length).toEqual(1);
-    expect(app.map(line => line.match(/        new GoodCSharpStack\(app, "GoodCSharp", new StackProps/)).filter(line => line).length).toEqual(1);
+    expect(app.map(line => line.match(/        new GoodCSharp\(app, "GoodCSharp", new GoodCSharpProps/)).filter(line => line).length).toEqual(1);
```

Comment:
I don't think this line should be changing here.

Reply:
Yup I'll fix the init template and change this

Reply:
There seems to have been a misunderstanding between us on how you wanted this to look. Lets sync offline on how we want the csharp applciation to look